service/src/main/kotlin/io/provenance/explorer/service/async/AsyncService.kt

```diff
@@ -146,4 +148,14 @@ class AsyncService(
     fun updateSpotlight() = transaction {
         explorerService.createSpotlight()
     }
+
+    @Scheduled(cron = "0 0/5 * * * ?") // Every 1 minute
+    fun retryBlockTxs() {
```

Comment:
Is this function used?

Reply:
Its on a schedule, so yes, its called asynchronously

Reply:
Ah, that makes sense